### ISSUE TITLE
Is it necessary to warm up model in the beginning of the training?

### ISSUE THREAD

**Issue by NLCharles** [open-mmlab/OpenPCDet#344]
Dear all,
I was using 8 gpus to train the model, and it seemed that  the warm-up training took forever to do. Is this really necessary?
In line 140 of https://github.com/open-mmlab/OpenPCDet/blob/master/tools/train.py
``` 
    model.train()  # before wrap to DistributedDataParallel to support fixed some parameters

```
